Syntax error on Automatus when running in python2

```
../../tests/unit/ssg_test_suite/test_rule.py:4: in <module>
    from ssg_test_suite import rule
E     File "/root/rpmbuild/BUILD/scap-security-guide-0.1.63/tests/ssg_test_suite/rule.py", line 91
E       f"Rule '{rule_name}' test setup script '{script}' "
E                                                         ^
E   SyntaxError: invalid syntax
```

Python 2 does not support the `format` string

https://github.com/ComplianceAsCode/content/blob/243c3a7da7bfb0d3bd78e51e2690ec3789d7de48/tests/ssg_test_suite/rule.py#L91

--- a/tests/ssg_test_suite/rule.py
+++ b/tests/ssg_test_suite/rule.py
@@ -88,8 +88,8 @@ def _apply_script(rule_dir, test_env, script):
 
         try:
             error_msg_template = (
-                f"Rule '{rule_name}' test setup script '{script}' "
-                "failed with exit code {rc}"
+                "Rule '{0}' test setup script '{1}' "
+                "failed with exit code {2}".format(rule_name, script, rc)
             )
             test_env.execute_ssh_command(
                 command, log_file, error_msg_template=error_msg_template)

